.github/workflows/gramine-report.yml

```diff
@@ -0,0 +1,63 @@
+name: generate a gramine signature, which can be verified later
+
+on:
+  workflow_call:
```

Comment:
is there a reason why we making this modular to be called from another workflow, instead of everything in one workflow file?